Source/engine/render/cel_render.hpp

```diff
@@ -20,6 +20,18 @@ namespace devilution {
  */
 std::pair<int, int> MeasureSolidHorizontalBounds(CelSprite cel, int frame = 0);
 
+/**
+ * @brief Apply the color swaps to a CEL sprite
+ *
+ * Only works on CEL files with frame headers.
+ *
+ * @param p CEL buffer
+ * @param translation Palette translation table
+ * @param numFrames Number of frames in the CEL file.
+ * @param hasFrameHeaders Whether the CEL has frame headers.

```

Comment:
It doesn't have these params and the description contradicts the last one.

Reply:
Fixed